components/nimbus/ios/Nimbus/FeatureVariables.swift

```diff
@@ -29,23 +29,39 @@ public protocol Variables {
     /// N.B. the `key` and type `String` should be listed in the experiment manifest.
     func getString(_ key: String) -> String?
 
+    func getStringList(_ key: String) -> [String]?
```

Comment:
Do we want to add the docs for each of those like we did in kotlin?

Reply:
Rats, I thought I'd done that!